src/components/form/Form.tsx

```diff
@@ -25,9 +31,16 @@ const Form = () => {
 
     localStorage.setItem("formData", JSON.stringify(plantsCollection));
 
+    toast.success("Roślinka dodana do kolekcji!");
+
     reset();
   };
 
+  errors.plantName?.type === "required" &&
+    toast.warning(errors.plantName?.message);
```

Comment:
I'd rather see those toast in the submit handler or even as a separate expression. If you want live validation you can change form mode to `onChange`